src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const price = days * 40;
```

Comment:
don't use magic numbers create constants for 40, 20, 50 ...etc and use them istead